website/content/en/docs/best-practices/observability-best-practices.md

```diff
@@ -65,6 +65,23 @@ The most common types are:
  - `Gauge` Value can be increased and decreased as needed.
 5. `_total` suffix should be used for accumulating count. If your metrics has labels with high cardinality, like `pod`/`container` it usually means that you can aggregate it more, thus it will not require `_total` suffix.
 
+#### Recommended Metrics
+
+* `operator_health_status` - The metric reports the operator health status. It should be reported by the operator itself based on the internal operator validations.  
+  This metric allows the cluster users, to view all operators health statuses with a single metric and improves the overall cluster observability.
+
+  In order to maintain consistency, Please use the following health statuses as the metric values:
+  * `0` - Indicates that the operator is healthy and working as expected.
+  * `1` - Indicates that the operator has some issues that needs to be addressed and can potentially lead to loss of functionality.
+  * `2` - Indicates that the operator is unhealthy and there is a loss of functionality that should be addressed.
+
+  Metric labels:
+  * `name` - The operator name.[Required].
+  * `reason` - The reason for the health issue, if possible.[Optional].
+
+  **Note:** In order for this metric to be useful for the users, it MUST be reported in a consistent way across all the operators.  
+  The metric values should be reported as numbers that represent the health statuses, since using strings like: Healthy/Warning/Error, can be implemented in many ways that would cause consistency issues and disagreements, like terminology preferences (For example: `Error` vs. `Unhealthy`), spelling errors, capital letters etc.
```

Comment:
```suggestion
  **Note:** In order for this metric to be useful for the users, it MUST be reported in a consistent way across all the operators. If you would prefer to follow any other level of granularity to indicate health statuses, it should be clearly documented such that an external observer can infer the indicator values accordingly. 
  
It is suggested to report the metric values as numbers that represent the health statuses, since using strings like: Healthy/Warning/Error, can be implemented in many ways that would cause consistency issues and disagreements, like terminology preferences (For example: `Error` vs. `Unhealthy`), spelling errors, capital letters etc.
```

WDYT? This is to ensure that we list out the best practice but not mention that this the only right approach since this doc is upstream. In the downstream documentation we can use the previous version and generalize on the set of indicators based on any further discussion if needed.

Reply:
Updated